core/src/main/java/com/scalar/db/transaction/consensuscommit/Snapshot.java

```diff
@@ -519,6 +519,10 @@ public boolean isValidationRequired() {
     return isExtraReadEnabled();
   }
 
+  public Optional<TransactionResult> getFromReadSet(Key key) {
```

Comment:
Super minor, but I think it's better to be placed below `containsKeyInReadSet` just to cluster similar methods.

Reply:
Indeed, thank you. I updated it in https://github.com/scalar-labs/scalardb/pull/899/commits/c61274026128d88a30b053d4722613253bc33d10